packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -0,0 +1,509 @@
+      }
+    },
+    '& .MuiTab-labelContainer': {
+      fontSize: '1rem'
+    }
+  },
+  contentGivenImage: {
+    marginTop: 185,
+    [theme.breakpoints.down('sm')]: {
+      marginTop: 130,
+    },
+  },
+  imageContainer: {
+    position: 'absolute',
+    [theme.breakpoints.down('sm')]: {
+      '& > img': {
+        height: 270,
+        width: '100%',
+      },
+      top: 77,
+      left: -4,
+    },
+    [theme.breakpoints.up('sm')]: {
+      top: 90,
+      '& > img': {
+        height: 300,
+        objectFit: 'cover',
+        width: '100%',
+      },
+    }
+  },
+  centralColumn: {
+    marginLeft: "auto",
+    marginRight: "auto",
+    maxWidth: MAX_COLUMN_WIDTH,
+  },
+  header: {
+    paddingTop: 19,
+    paddingBottom: 0,
+    paddingLeft: 42,
+    paddingRight: 42,
+    background: theme.palette.panelBackground.default,
+    width: "100%",
+  },
+  titleRow: {
+    [theme.breakpoints.up('sm')]: {
+      display: 'flex',
+      justifyContent: 'space-between',
+    }
+  },
+  title: {
+    ...theme.typography.display3,
+    ...theme.typography.commentStyle,
+    marginTop: 0,
+    fontWeight: 600,
+    fontVariant: "small-caps"
+  },
+  notifyMeButton: {
+    [theme.breakpoints.down('xs')]: {
+      marginTop: 6,
+    },
+  },
+  editMenu: {
+    [theme.breakpoints.down('xs')]: {
+      marginTop: 16,
+      marginBottom: 8,
+    },
+  },
+  wikiSection: {
+    paddingTop: 12,
+    paddingLeft: 42,
+    paddingRight: 42,
+    paddingBottom: 12,
+    marginBottom: 24,
+    background: theme.palette.panelBackground.default,
+  },
+  subHeading: {
+    paddingLeft: 42,
+    paddingRight: 42,
+    marginTop: -2,
+    background: theme.palette.panelBackground.default,
+    ...theme.typography.body2,
+    ...theme.typography.postStyle,
+  },
+  subHeadingInner: {
+    paddingTop: 2,
+    paddingBottom: 2,
+    borderTop: theme.palette.border.extraFaint,
+    borderBottom: theme.palette.border.extraFaint,
+  },
+  relatedTagLink : {
+    color: theme.palette.lwTertiary.dark
+  },
+  tagHeader: {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+  },
+  postsTaggedTitle: {
+    color: theme.palette.grey[600]
+  },
+  pastRevisionNotice: {
+    ...theme.typography.commentStyle,
+    fontStyle: 'italic'
+  },
+  nextLink: {
+    ...theme.typography.commentStyle
+  },
+  sidebarBoxWrapper: {
+    backgroundColor: theme.palette.panelBackground.default,
+    border: theme.palette.border.commentBorder,
+    padding: "0em 1.5em",
+  },
+  tableOfContentsWrapper: {
+    padding: 24,
+  }
+});
+
+export const tagPostTerms = (tag: TagBasicInfo | null, query: any) => {
+  if (!tag) return
+  return ({
+    ...query,
+    filterSettings: {tags:[{tagId: tag._id, tagName: tag.name, filterMode: "Required"}]},
+    view: "tagRelevance",
+    tagId: tag._id,
+  })
+}
+
+// TODO rename
+const subforumTabs = ["posts", "wiki"] as const
```

Comment:
Nit: This could be a good use case for [TupleSet](https://cea-core.slack.com/archives/C75BWSNBH/p1668556264046269)?